### PR TITLE
feat: added login and signup page

### PR DESCRIPTION
**Pull request by arjun-computer-geek**
added signup and login page

### CODE REVIEW

File: css/login.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    box-shadow: 0 0 20px 0 rgba(0, 0, 0, 0.1);
+    overflow: hidden;
+}
+ 
+.login-main .form-container span {
+    font-weight: bold;
+    padding: 0 .5em;
+    color: #555;
+    cursor: pointer;
+    width: 6.25rem;
```
Comment (Rockyraven) [inline css/login.css:23]: use var color

File: css/login.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
+@import url('./style.css');
+.login-main{
+    padding-top: 2rem;
+}
+.login-main .form-container {
+    margin-top: 10rem;
+    background: #fff;
```
Comment (Developers-codz) [inline css/login.css]: plz use variables

**Review by Rockyraven — APPROVED** [review]
all goods for me can merge

File: login.html [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+            </a>
+        </div>
+    </nav>
+
+    <!-- main content -->
+    <main class="login-main">
+        <div class="form-container">
+            <div class="form-btn">
+                <span onclick="register()">Register</span>
+                <span onclick="login()">Login</span>
```
Comment (Developers-codz) [inline login.html]: why JS in html file??

**Review by Developers-codz — APPROVED** [review]
Else Look good to me

File: css/login.css (hunk 1 above)
Comment (arjun-computer-geek) [reply css/login.css:23]: it left by mistake

File: css/login.css (hunk 2 above)
Comment (arjun-computer-geek) [reply css/login.css]: ya left by mistake

**Review by vishalg8454 — APPROVED** [review]

**Pull request merged**